src/devices/Bmxx80/Bmxx80.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <EnableDefaultItems>false</EnableDefaultItems>
     <LangVersion>8.0</LangVersion>
+    <Configurations>Windows-Debug;Linux-Debug;Linux-Release;Windows-Release;Debug;Release</Configurations>
```

Comment:
Same comment here and in the SoftwareSpi project around removing configurations and ProjectReference conditions.